src/main/java/com/tenx/ms/retail/order/service/OrderService.java

```diff
@@ -0,0 +1,137 @@
+package com.tenx.ms.retail.order.service;
+
+import com.tenx.ms.retail.client.domain.ClientEntity;
+import com.tenx.ms.retail.client.repository.ClientRepository;
+import com.tenx.ms.retail.order.domain.OrderDetailEntity;
+import com.tenx.ms.retail.order.domain.OrderEntity;
+import com.tenx.ms.retail.order.repository.OrderDetailRepository;
+import com.tenx.ms.retail.order.repository.OrderRepository;
+import com.tenx.ms.retail.order.rest.dto.Order;
+import com.tenx.ms.retail.order.rest.dto.OrderDetail;
+import com.tenx.ms.retail.order.util.OrderConverter;
+import com.tenx.ms.retail.order.util.OrderDetailStatusEnum;
+import com.tenx.ms.retail.product.domain.ProductEntity;
+import com.tenx.ms.retail.product.repository.ProductRepository;
+import com.tenx.ms.retail.stock.domain.StockEntity;
+import com.tenx.ms.retail.stock.repository.StockRepository;
+import com.tenx.ms.retail.store.domain.StoreEntity;
+import com.tenx.ms.retail.store.repository.StoreRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.stereotype.Service;
+
+import javax.validation.ConstraintViolationException;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+
+/**
+ * Created by goropeza on 28/08/16.
+ */
+@Service
+public class OrderService {
+    @Autowired
+    private OrderConverter orderConverter;
+    @Autowired
+    private OrderRepository orderRepository;
+    @Autowired
+    private OrderDetailRepository orderDetailRepository;
+    @Autowired
+    private ClientRepository clientRepository;
+    @Autowired
+    private StoreRepository storeRepository;
+    @Autowired
+    private StockRepository stockRepository;
+    @Autowired
+    private ProductRepository productRepository;
+
+    @SuppressWarnings("PMD")
```

Comment:
Recommend to suppress individual PMD violations if those violations really cannot be fixed. Suppressing all PMD all any classes or methods except test cases is not good as you'll miss any future PMD violations when you modify these classes or methods.